lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -856,10 +855,18 @@ void SessionState::get_session_info(SessionState::SessionInfo& info) {
   info.ipv6_addr = config_.common_context.ue_ipv6();
   info.teids     = config_.common_context.teids();
   info.msisdn    = config_.common_context.msisdn();
-  get_dynamic_rules().get_rules(info.dynamic_rules);
-  get_gy_dynamic_rules().get_rules(info.gy_dynamic_rules);
+  info.ambr      = config_.get_apn_ambr();
+
+  dynamic_rules_.get_rules(info.gx_dynamic_rules);
+  gy_dynamic_rules_.get_rules(info.gy_dynamic_rules);
   info.static_rules = active_static_rules_;
-  info.ambr         = config_.get_apn_ambr();
+
+  for (const std::string& rule_id : active_static_rules_) {
+    PolicyRule rule;
+    if (static_rules_.get_rule(rule_id, &rule)) {
+      info.gx_static_rules.push_back(rule);
+    }
+  }
```

Comment:
I don't remember all the usages of `get_session_info` but shouln't we keep the concept of static and dynamic on sessiond? and only remove it once we talk to pipelined?

Reply:
SessionInfo is only used in terms of communicating to PipelineD, so I think we're OK there. From a quick glance it is used in AAAClient, but we should migrate that to use `session->get_config` anyways.